src/main/java/youyihj/zenutils/api/item/CrTFluidHandler.java

```diff
@@ -0,0 +1,35 @@
+package youyihj.zenutils.api.item;
+
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.liquid.ILiquidStack;
+import crafttweaker.api.minecraft.CraftTweakerMC;
+import net.minecraftforge.fluids.capability.IFluidHandler;
+import stanhebben.zenscript.annotations.IterableSimple;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+import java.util.Objects;
+
+/**
+ * @author various.authors
+ */
+@ZenClass("mods.zenutils.ItemHandler")
+@IterableSimple("crafttweaker.item.IItemStack")
```

Comment:
https://docs.blamejared.com/1.12/en/Dev_Area/ZenAnnotations/Annotation_Iterable
So the class should implement `Iterable<IItemStack>`. But this is a fluid handler, you should change it to `ILiquidStack` or other else about fluids?

Reply:
Should this class implement Iterable?

Reply:
Yes, or just remove the annotation.

Reply:
I'm just going to remove the annotation because I don't really understand iterable.